.github/workflows/build.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
   test:
     # When running with act (https://github.com/nektos/act), these lines need to be appended with the ACT variable
     # to force each job to run
-    if: github.repository == 'adobe/buildrunner'
+    if: github.repository == 'adobe/buildrunner' || ${{ env.ACT }}
```

Comment:
This will not build on github if I remember right.

Reply:
Yup: https://github.com/adobe/buildrunner/actions/runs/2157237929

Don't know why, but that's how it works.